src/AnimatedRegion.ts

```diff
@@ -60,10 +73,10 @@ export default class AnimatedMapRegion extends AnimatedWithChildren {
   }
 
   setValue(value: Region) {
-    this.latitude._value = value.latitude;
-    this.longitude._value = value.longitude;
-    this.latitudeDelta._value = value.latitudeDelta;
-    this.longitudeDelta._value = value.longitudeDelta;
+    this.latitude.setValue(value.latitude);
+    this.longitude.setValue(value.latitude);
```

Comment:
Is change `value.longitude` -> `value.latitude`  intentional ?

Reply:
No it was a typo, already fixed, thanks